ci/git-branch-or-tag.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
```

Comment:
I am not very proud of the duplication in the two shell scripts, open to suggestions to reduce duplication, if done in a way that doesn't add complexity.

Reply:
You could replace the `echo` with two functions that would echo their argument or nothing for one and its argument or "TRUE" for the other based on an argument to the script. The script could be called `git-branch-tag-or-release.sh`.

Reply:
I was thinking along these lines, but I was cooked last week. I will give it a try.